frontend/src/components/App/FishbowlInfo/styles.ts

```diff
@@ -63,18 +63,28 @@ const Icon = styled.button`
 `;
 
 const Description = styled.div`
+  position: fixed;
+  top: ${space(15)};
+  left: 50%;
+  transform: translateX(-50%);
+  width: calc(100% - ${space(4)});
+  padding: ${space(2)};
   background-color: ${COLOR_NEUTRO_100};
   border: 1px solid ${COLOR_NEUTRO_700};
   border-radius: ${BORDER_RADIUS};
-  box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.1);
+  box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.15);
   color: ${COLOR_NEUTRO_700};
```

Comment:
As a convention, we tend to order CSS properties alphabetically

Reply:
👍🏽  okay I thought we used idiomaticCss. Should we add stylelint?